docs/dev/plugins/bash-script-generator/bash-script-generator.md

```diff
@@ -20,10 +23,13 @@ function check_executable () {
 ```
 
 **Java wrapper:**
+
 ```Java
 public void checkExecutable (String executable) {
     list.add("check_executable " + executable);
 }
 ```
 
 A plugin can extend the `BashScriptGenerator` if it has additional functions.
+
+In general the scripts are saved in the `output\scripts\utils\` folder in the target artifact. If a plugin provides is own utils functions the script containing them is saved with the name `utils-<plugin-name>.sh`.
```

Comment:
Maybe use forward slashes to denote paths.
Maybe rename `utils` to `util`.
One would not use the `pluginName`, but the `pluginId` to describe the name of the script.

Please also add the changes to the target artifact convention docs.